src/index.ts

```diff
@@ -270,6 +289,11 @@ export class Datastore {
 
         let storageId = ""
         let file = await this.getFile(fileId)
+        if (JSZip.support.arraybuffer) {
```

Comment:
Not exactly sure why we're doing this?

Reply:
`getFile()` returns the file uncompressed. We need to re-compress it.